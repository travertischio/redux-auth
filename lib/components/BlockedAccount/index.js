"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _react = require("react");

var _react2 = _interopRequireDefault(_react);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var BlockedAccount = function BlockedAccount() {
  return _react2.default.createElement(
    "div",
    { id: "blocked-account" },
    _react2.default.createElement(
      "h1",
      null,
      "Your account has been locked because of too many failed login attempts. Please contact your administrator."
    )
  );
}; /**
   *
   * BlockedAccount
   *
   */

exports.default = BlockedAccount;