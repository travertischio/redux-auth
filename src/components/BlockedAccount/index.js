/**
*
* BlockedAccount
*
*/

import React from 'react';

const BlockedAccount = () => (
  <div id="blocked-account">
    <h1>Your account has been locked because of too many failed login attempts. Please contact your administrator.</h1>
  </div>
);

export default BlockedAccount;
