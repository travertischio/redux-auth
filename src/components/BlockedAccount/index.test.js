import React from 'react';
import { createComponentWithIntl } from 'react-unit-testing-utils';
import BlockedAccount from './';

describe('<BlockedAccount />', () => {
  it('should render BlockedAccount', () => {
    const initialState = {};
    const { component } = createComponentWithIntl(<BlockedAccount />, initialState);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
