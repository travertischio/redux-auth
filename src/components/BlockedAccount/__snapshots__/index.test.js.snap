// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockedAccount /> should render BlockedAccount 1`] = `
<div
  id="blocked-account"
>
  <h1>
    Your account has been locked because of too many failed login attempts. Please contact your administrator.
  </h1>
</div>
`;
